.github/workflows/release.yml

```diff
@@ -20,13 +20,18 @@ jobs:
 
       - name: Build from source
         run: |
-          CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-s -w" -o ./bin/wait-for-it
+          CGO_ENABLED=0 GOOS=linux GOARCH=386 go build -ldflags "-s -w" -o ./bin/wait-for-it.linux-386.exe
+          CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-s -w" -o ./bin/wait-for-it-linux-amd64.exe
+          CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -ldflags "-s -w" -o ./bin/wait-for-it-linux-arm64.exe
         
-      - name: Upload artifact
+      - name: Upload artifacts
         uses: actions/upload-artifact@v2
         with:
-          name: wait-for-it
-          path: ./bin/wait-for-it
+          name: linux artifacts
+          path: |
+            ./bin/wait-for-it.linux-i386 
+            ./bin/wait-for-it.linux-amd64
+            ./bin/wait-for-it.linux-arm64
```

Comment:
I think we should also copy the `wait-for-it.linux-arm64` binary to `wait-for-it`, since I've seen a couple of people use it in their docker containers directly from the releases on this repo. It would break for them if we don't have a binary called `wait-for-it`.